inc/composer/class-command.php

```diff
@@ -492,7 +493,30 @@ protected function status( InputInterface $input, OutputInterface $output ) {
 	 * @return int
 	 */
 	protected function logs( InputInterface $input, OutputInterface $output ) {
-		$log = $input->getArgument( 'options' )[0];
+		var_dump( 'start' );
+		if( !isset( $input->getArgument( 'options' )[0] ) ){
+			$helper = $this->getHelper( 'question' );
+			$question = new ChoiceQuestion(
+				'Please select your prefered service (defaults to php)',
```

Comment:
Some slight wording changes here:
```suggestion
				'Please select a service (defaults to php)',
```